Description: Put all orphan files into the airporthangup package
 Some files were not declared as being part of a package.
Author: Julien BLACHE <jblache@debian.org>

Index: airport-utils-2/modem/AirportBaseStationHangup.java
===================================================================
--- airport-utils-2.orig/modem/AirportBaseStationHangup.java	2005-03-01 02:30:27.000000000 +0100
+++ airport-utils-2/modem/AirportBaseStationHangup.java	2010-03-11 20:56:30.819753303 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthangup;
 
 import java.util.*;
 import java.net.*;
Index: airport-utils-2/modem/AirportBaseStationHangupNoSwing.java
===================================================================
--- airport-utils-2.orig/modem/AirportBaseStationHangupNoSwing.java	2005-03-01 00:35:29.000000000 +0100
+++ airport-utils-2/modem/AirportBaseStationHangupNoSwing.java	2010-03-11 20:56:30.832915916 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthangup;
 
 import java.net.*;
 import java.awt.*;
Index: airport-utils-2/modem/DialogResponse.java
===================================================================
--- airport-utils-2.orig/modem/DialogResponse.java	2006-04-17 04:19:54.000000000 +0200
+++ airport-utils-2/modem/DialogResponse.java	2010-03-11 20:56:30.866336041 +0100
@@ -19,6 +19,8 @@
  *
  */
 
+package airporthangup;
+
 public class DialogResponse
 {
 	public Boolean cancelled;
Index: airport-utils-2/modem/InputDialog.java
===================================================================
--- airport-utils-2.orig/modem/InputDialog.java	2006-04-17 04:20:00.000000000 +0200
+++ airport-utils-2/modem/InputDialog.java	2010-03-11 20:56:30.891418091 +0100
@@ -19,11 +19,12 @@
  *
  */
 
+package airporthangup;
+
 import java.awt.*;
 import java.awt.event.*;
 
 
-
 class InputDialog extends Dialog
 					implements ActionListener
 {
Index: airport-utils-2/modem/MessageDialog.java
===================================================================
--- airport-utils-2.orig/modem/MessageDialog.java	2006-04-17 04:20:05.000000000 +0200
+++ airport-utils-2/modem/MessageDialog.java	2010-03-11 20:56:30.916503981 +0100
@@ -19,11 +19,12 @@
  *
  */
 
+package airporthangup;
+
 import java.awt.*;
 import java.awt.event.*;
 
 
-
 class MessageDialog extends Dialog
 					implements ActionListener
 {
Index: airport-utils-2/modem/OSUNMSInfoElement.java
===================================================================
--- airport-utils-2.orig/modem/OSUNMSInfoElement.java	2005-03-01 00:35:55.000000000 +0100
+++ airport-utils-2/modem/OSUNMSInfoElement.java	2010-03-11 20:56:30.941752288 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthangup;
 
 
 
Index: airport-utils-2/modem/OSUNMSInfoHash.java
===================================================================
--- airport-utils-2.orig/modem/OSUNMSInfoHash.java	2002-06-18 17:12:07.000000000 +0200
+++ airport-utils-2/modem/OSUNMSInfoHash.java	2010-03-11 20:56:30.966752180 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthangup;
 
 import java.util.*;
 import java.io.*;
